doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
+| ROS2    | Type    | Namespace::Topic | -                      |
+| AUTOSAR | Service | Instance         | Event                  |
+| DDS     | -       | -                | /Group/Instance/Topic  |
 
-##### 2. Functional approach with loaning
+Service is related to instance like classes are related to objects in C++. Service describes an abstract topic and an 
+instance is one instantiation of that abstraction. Like an object is an instantiated class. Events are in this context 
+like members of a class.
 
+Example:
 ```cpp
-publisher.loan(sizeof(CounterTopic))
-    .and_then([&](iox::popo::Sample<void>& sample) {
-        new (sample.get()) CounterTopic(73);
-        sample.publish();
-    })
-    .or_else([](iox::popo::AllocationError) {
-        /* handle the error */
-    });
-```
+class MyRadarService {
+   public:
+      bool hasObstacleDetected;
+      float distanceToObstacle;
+};
 
-Notice that we get an untyped sample, ``iox::popo::Sample<const void>``. We could also use ``auto& sample`` in the
-lambda arguments to shorten it. Again we access the pointer to the underlying raw memory of the sample and
-construct the data we want to send.
+MyRadarService frontLeftRadarInstance;
+std::cout << frontLeftRadarInstance.hasObstacleDetected << std::endl;
+```
+In the iceoryx world we would subscribe to the service ``("MyRadarService", "frontLeftRadarInstance", "hasObstacleDetected")`` 
+and would receive a sample whenever an obstacle was detected. Or we would subscribe to ``distanceToObstacle`` and would 
+receive a constant stream of data which presents the distance to the obstacle.
 
-#### Creating a subscriber
+### Restrictions
 
-While the string identifiers still have to match those in the publisher, as in the untyped publisher there is no
-template type argument.
+The data type of the transmitted data can be any C++ class, struct or plain old data type as long as it satisfies the 
+following conditions:
+- no heap is used
+- the data structure is entirely contained in the shared memory - no pointers to process local memory, no references 
+to process local constructs, no dynamic allocators
+- the data structure has to be relocatable and therefore must not internally use pointers/references
+- no virtual members
 
-```cpp
-iox::popo::UntypedSubscriber subscriber({"Group", "Instance", "CounterTopic"});
-subscriber.subscribe();
-```
+!!! note
+    Most of the STL types cannot be used, but we reimplemented some of them so that they meet the conditions above. 
+    You can find an overview [here](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_utils#cxx).
 
-#### Receiving data
+## Publisher
 
-Receiving the data works in the same way as in the typed API, the main difference is the ``reinterpret_cast`` needed
-before accessing the data (since the subscriber has no knowledge of the underlying type).
+A publisher is tied to a topic and needs a service description to be constructed. If it is typed, one needs to
```

Comment:
`icehello` from #655 only uses the typed API. I would leave out the untyped or C API at this point and just answer the questions "What is a publisher?" "How is it related to the other building blocks in iceoryx that are depicted on the graphic?"

The difference between untyped, typed is introduced in the icedelivery example.

Reply:
In my opinion it belongs to an overview.

Reply:
Ok, should I then remove the explanation in the icedelivery example or is ok from your side if we explain it again there?

Reply:
It should be explained in the example too